<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task</class>
 <widget class="QDialog" name="Task">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #E1E7F2;
}

QLabel {
    color: #000;
    font-weight: bold;
    font-size: 14px;
}

QCheckBox[objectName=&quot;documentationBox&quot;] {
    font-weight: bold;
}

QPushButton {
    padding: 10px;
    color: white;
    font-weight: bold;
    border: none;
}

/* default button style */
QPushButton {
    background-color: black;
}

QPushButton:hover {
    background-color: grey;
}

/* OKbutton button style */
QPushButton[objectName=&quot;OKbutton&quot;] {
    background-color: #85B404;
}

QPushButton[objectName=&quot;OKbutton&quot;]:hover {
    background-color: #A2DC04;
}

/* refresh button style */
QPushButton[objectName=&quot;markButton&quot;], QPushButton[objectName=&quot;fileButton&quot;] {
    background-color: #197BBD;
}

QPushButton[objectName=&quot;markButton&quot;]:hover {
    background-color: #6A7FDB;
}

QPushButton[objectName=&quot;markButton&quot;]:disabled {
    background-color: #87898D;
}

QPushButton[objectName=&quot;fileButton&quot;]:hover {
    background-color: #6A7FDB;
}

/* &quot;deny&quot; button style */
QPushButton[objectName=&quot;returnButton&quot;] {
    background-color: #E0001E;
}

QPushButton[objectName=&quot;returnButton&quot;]:hover {
    background-color: #FF0022;
}

QGroupBox[objectName=&quot;groupboxMarks&quot;] {
    border: none;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="14" column="0">
    <widget class="QGroupBox" name="groupboxMarks">
     <layout class="QGridLayout" name="markLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <widget class="QTextEdit" name="errors">
        <property name="toolTip">
         <string notr="true"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="percentageSign">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLCDNumber" name="percentageMark">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>% success</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="digitCount">
         <number>3</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Filled</enum>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
        <property name="intValue" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QGridLayout" name="uploadLayout">
     <item row="2" column="2">
      <widget class="QPushButton" name="markButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Marks chosen solution</string>
       </property>
       <property name="text">
        <string>Mark</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <spacer name="spacerReturnMark">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="returnButton">
       <property name="text">
        <string>Return</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="fileButton">
       <property name="text">
        <string>Choose file</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2" colspan="2">
      <widget class="QLineEdit" name="fileName">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Chosen file for validationg</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QVBoxLayout" name="documentationLayout">
     <item>
      <widget class="QCheckBox" name="documentationBox">
       <property name="text">
        <string>Recommended recommendation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="documentationEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QVBoxLayout" name="taskLayout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="taskName">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="taskText">
       <property name="toolTip">
        <string>Writen task</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="Line" name="lineUploadScore">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBarSolution">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Progres of marking</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="Line" name="lineTaskUpload">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QVBoxLayout" name="runtimeExamplesLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QCheckBox" name="runtimeExamplesBox">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Runtime examples</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="runtimeExamplesEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
